lib/asset/getGamePassProductInfo.js

```diff
@@ -0,0 +1,51 @@
+// Includes
+const http = require('../util/http.js').func
+const cache = require('../cache')
+
+// Args
+exports.required = ['gamepass']
+
+// Docs
+/**
+ * ✅ Get the info of an gamepass.
+ * @category Assets
+ * @param {number} gamePassId - The id of the asset.
+ * @returns {Promise<GamePassProductInfo>}
+ * @example const noblox = require("noblox.js")
+ * const gamePassInfo = await noblox.getGamePassProductInfo(2919875)
+**/
+
+// Define
+function getGamePassProductInfo (gamepass) {
+  return new Promise((resolve, reject) => {
+    const httpOpt = {
+      url: `//apis.roblox.com/game-passes/v1/game-passes/${gamepass}/product-info`,
```

Comment:
is there any other documented endpoint for this?

Reply:
This is what Roblox staff recommend, I'm unaware of any other endpoints outside of the economy api (which is also undocumented)
https://devforum.roblox.com/t/action-needed-sunsetting-apirobloxcom/1796897#marketplace-5